test/e2e/framework/service_util.go

```diff
@@ -874,13 +1039,12 @@ func newEchoServerPodSpec(podName string) *v1.Pod {
 	return pod
 }
 
-// LaunchEchoserverPodOnNode launches a pod serving http on port 8080 to act
+// LaunchEchoserverPod launches a pod serving http on port 8080 to act
 // as the target for source IP preservation test. The client's source ip would
 // be echoed back by the web server.
-func (j *ServiceTestJig) LaunchEchoserverPodOnNode(f *Framework, nodeName, podName string) {
```

Comment:
What's the reason for removing the OnNode part?

Reply:
Scheduling on specific node was not having significant importance [here](https://github.com/kubernetes/kubernetes/pull/79667/files/3ff4ec6020dd1db4265a293a8cb9cc713f515b04#diff-3e87abdcaa68e4594b468bc5e5d99551L306) so thought to relax this constraint. Also there were no other places where this function is used so thought to change method itself. 

I understand, passing "" to nodeName could have schedule the pod to any node and changing `_util` functions are not good idea but having only 1 reference to this method made me feel to make this change.

Reply:
Should I revert this change or it's ok in this case?

Reply:
Seems fine to keep, thanks for the context